OvmfPkg/OvmfPkgX64.dsc

```diff
@@ -28,11 +28,20 @@
   # Defines for default states.  These can be changed on the command line.
   # -D FLAG=VALUE
   #
-  DEFINE SECURE_BOOT_ENABLE      = FALSE
-  DEFINE SMM_REQUIRE             = FALSE
+  DEFINE SECURE_BOOT_ENABLE      = TRUE
+  DEFINE SMM_REQUIRE             = TRUE
   DEFINE SOURCE_DEBUG_ENABLE     = FALSE
-  DEFINE TPM_ENABLE              = FALSE
-  DEFINE TPM_CONFIG_ENABLE       = FALSE
+  DEFINE TPM_ENABLE              = TRUE
+  DEFINE TPM_CONFIG_ENABLE       = TRUE
+  DEFINE SATA_PASSWORD_ENABLE    = TRUE
+  DEFINE OPAL_PASSWORD_ENABLE    = TRUE
+  DEFINE LOAD_OPTION_ROMS        = FLASE

```

Comment:
This will not be used by QEMU/OVMF, please remove

Reply:
Should I remove the LOAD_OPTIONS_ROMS line or all the lines?

Reply:
I have marked only the LOAD_OPTION_ROMS, but GItHub lieks to preview a little bit more. In short:

```suggestion
```